hooks/601-clean-cracklib.chroot

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
```

Comment:
actually one quick comment - maybe a small readme why we remove most of the package, something like: `// we only need the cracklib_dict.pwd so that pamXXX works correctly, see LP:423423`

Reply:
Done.